src/main/java/com/attacktimer/AttackTimerMetronomePlugin.java

```diff
@@ -408,19 +412,26 @@ private int getWeaponSpeed(int weaponId, Integer firedProjectile, PoweredStaves
         return adjustSpeedForLeaguesIfApplicable(speed); // Deadline for next available attack.
     }
 
+    private static final int TARGET_DUMMY_ID = 10507;
+
     private boolean isPlayerAttacking()
     {
         int animationId = client.getLocalPlayer().getAnimation();
+        if (AnimationData.isBlockListAnimation(animationId))
+        {
+            return false;
+        }
         // Testing if we are attacking by checking the target is more future
         // proof to new weapons which don't need custom code and the weapon
         // stats are enough.
         Actor target = client.getLocalPlayer().getInteracting();
         if (target != null && (target instanceof NPC))
```

Comment:
Does this break the timer for PvP?

Reply:
Doh, NGL I didn't test PvP it's something I rarely do. I'll test it today and make fixes if it's required.